addons/im_livechat/demo/im_livechat_channel/im_livechat_session_2.xml

```diff
@@ -15,6 +15,8 @@
             <field name="partner_id" ref="base.partner_demo"/>
             <field name="livechat_member_type">agent</field>
             <field name="create_date" eval="DateTime.today() + relativedelta(months=-1, days=-1)"/>
+            <field name="response_time_hour" eval="8/60"/>
+            <field name="message_count">2</field>
```

Comment:
It always shows as 1 in the view? Is it because average is actually just 1 for all users?

![image](https://github.com/user-attachments/assets/846d9876-e5f3-4ea9-963e-1e8fe8b3c90b)